src/mono/browser/browser.proj

```diff
@@ -16,7 +16,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <MonoWasmThreads Condition="'$(MonoWasmBuildVariant)' == 'multithread'">true</MonoWasmThreads>
+    <MonoWasmThreads Condition="'$(WasmEnableThreads)' == 'true'">true</MonoWasmThreads>
```

Comment:
Do we need a diffrent property?

Reply:
This is also public unfortunately, used by developers:
https://github.com/dotnet/runtime/blob/cd460dbda99c080b9f9ec1cbdcd233f25ba8c189/src/mono/browser/browser.proj#L564C76-L565C77

Reply:
The `browser.proj` isn't shipped to the customers. So the refered developer is some how builds the runtime repo.